cypress/integration/page.spec.js

```diff
@@ -248,27 +277,27 @@
         cy.get('@comments').should('not.be.called');
       });
     });
-  })
+  });
 
   describe('UserSelector', () => {
     const { el, button, users, selectedUser } = userSelector;
 
     describe('', () => {
-      it('should have all the loaded users', () => {
+      it.skip('should have all the loaded users', () => {
         page.mockUsers();
         cy.visit('/');
 
         users().should('have.length', 10);
       });
 
-      it('should not have users hardcoded', () => {
-        cy.intercept('**/users', { fixture: 'someUsers' })
+      it.skip('should not have users hardcoded', () => {
```

Comment:
This test is skipped. Ensure that it is intentionally skipped and consider enabling it if it is relevant to the test suite.